src/html/m_span.cpp

```diff
@@ -38,6 +38,10 @@ TAG_HANDLER_BEGIN(SPAN, "SPAN" )
         int oldunderlined = m_WParser->GetFontUnderlined();
         wxString oldfontface = m_WParser->GetFontFace();
 
+        auto cell = new wxHtmlFontCell(m_WParser->CreateCurrentFont());
```

Comment:
It looks like this is the only place where a new cell is now being added, instead of just setting ID for an existing cell.

Could you please explain why is this necessary, especially when there is no ID at all?